ThunderCloud/CheckView.swift

```diff
@@ -100,21 +54,8 @@ class CheckView: UIControl {
 	
 	private func setup() {
 		
-		guard innerView == nil else { return }
-		
-		outerView = UIView(frame: bounds)
-		outerView?.cornerRadius = borderRadius
-		addSubview(outerView!)
-		
-		innerView = UIView(frame: bounds.insetBy(dx: 1.5, dy: 1.5))
-		innerView?.cornerRadius = borderRadius - 3
-		innerView?.backgroundColor = .white
-		addSubview(innerView!)
-		
-		onTintColor = ThemeManager.shared.theme.mainColor
-		if tintColor == nil {
-			tintColor = UIColor(red: 0.90, green: 0.90, blue: 0.90, alpha: 1.0)
-		}
+        imageView = UIImageView(image: (#imageLiteral(resourceName: "check-off") as StormImageLiteral).image)
+		addSubview(imageView!)
```

Comment:
What'd (potentially) be amazing is an extension on UIView like `addOptionalSubview()` where you can pass in any optional UIView and it'll attempt to add it, otherwise it'll ignore it - so we don't have to do the force unwrapping. What are your thoughts on that? :)

Reply:
I’ll just add an `if let` in for now, but I like that as a suggested addition to ThunderBasics!